tools/functional-tester/etcd-tester/lease_stresser.go

```diff
@@ -366,6 +368,8 @@ func (ls *leaseStresser) attachKeysWithLease(leaseID int64) error {
 		txn := &pb.TxnRequest{Success: txnPuts}
 		_, err := ls.kvc.Txn(ls.ctx, txn)
 		if err == nil {
+			// since all created keys will be deleted too, the number of operations on keys will be roughly 2x the number of created keys
```

Comment:
Huh? Since this is counting deletes as an operation as well, shouldn't `err := ls.rateLimiter.WaitN(ls.ctx, ls.numLeases*ls.keysPerLease)` in `leaseStresser.run` be changed to `err := ls.rateLimiter.WaitN(ls.ctx, ls.numLeases*2*ls.keysPerLease)` to account for that?